conformance-tests/Makefile

```diff
@@ -0,0 +1,52 @@
+GATEWAY_CLASS = nginx
+SUPPORTED_FEATURES = Gateway
+KIND_KUBE_CONFIG_FOLDER = $${HOME}/.kube/kind
+TAG = latest
+PREFIX = conformance-test-runner
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: Makefile ## Display this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: build-test-image
+build-test-image: ## Build conformance test image
+	docker build -t $(PREFIX):$(TAG) -f Dockerfile ..
+
+.PHONY: create-kind-cluster
+create-kind-cluster: ## Create a KinD cluster
+	kind create cluster --image kindest/node:v1.27.1
+	kind export kubeconfig --kubeconfig $(KIND_KUBE_CONFIG_FOLDER)/config
+
+.PHONY: install-nkg
+install-nkg: ## Install NKG with provisioner on configured KinD cluster
+	kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.0/standard-install.yaml
+	./wait_for_webhook.sh 
```

Comment:
Actually, I just remember the `kubectl wait` command.

I think you could do: `kubectl -n gateway-system wait --for=condition=available deploy gateway-api-admission-server`. Can also set a timeout value if desired, default is 30s.

I'm assuming that's the deployment you're waiting for.

Reply:
yep that seems to be working, ty